plugins/gt-workflow/skills/gt-cleanup/SKILL.md

```
@@ -526,11 +535,20 @@ If the user chooses "Yes", invoke the Skill tool with
 **Graceful degradation:** If the Skill call fails (yellow-core not installed or
 command not found), report:
 
+On Claude Code:
+
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)